tiledb/ml/models/tensorflow_keras.py

```diff
@@ -34,16 +34,22 @@ class TensorflowKerasTileDBModel(TileDBModel):
 
     def save(
         self,
-        include_optimizer: bool = False,
+        *,
         update: bool = False,
-        meta: Optional[dict] = {},
-    ):
+        meta: Optional[Meta] = None,
+        include_optimizer: bool = False,
+    ) -> None:
```

Comment:
I know that we are using this format in some functions but is there any way to have a unified way? There are functions that have the same number of args and are onliners and others that expand their args in different lines. Is `black` doing it or is there any way to impose a single function signature format?

Reply:
AFAICT it depends (only) on the length of the line; if it can fit all args in one line it does. Other than overriding the default [line-length](https://github.com/psf/black/blob/41e670064063e3e221b1c3c40db5aaae784b5231/pyproject.toml#L9) I doubt you can impose a format.